### ISSUE TITLE
Update to OCP v4.13 and remove FIPS

### ISSUE THREAD

**Issue by AdrianHammond** [finos/ansible-cfi-ocp#16]
### Description of Problem:
update OCP release to v4.13 and remove FIPS as no longer required by Service Accelerator

**Comment by johanalimka** [comment]
PR raised with the changes to reflect OCP 4.3 documentation and FIPS removed. Testing completed.

**Comment by AdrianHammond** [comment]
merge into dev - closing issue as now waiting on issue #25 to merge into Main.
